code/modules/mob/living/carbon/human/species_types/zombies.dm

```diff
@@ -127,6 +129,9 @@
 
 /// We start fading out the human and qdel them in set time
 /datum/species/zombie/proc/fade_out_and_qdel_in(mob/living/carbon/human/H, time = 5 SECONDS)
+	if(!perma)//Stops from having more than one stat per zombie
```

Comment:
This doesn't work at all, you're changing a singleton, this is shared between all zombies of the same species.